advancing-bitcoin/2020/2020-02-06-antoine-riard-taproot-lightning.es.md

```diff
@@ -1,20 +1,18 @@
 ---
-title: Antoine Riard - Taproot Lightning (2020-02-06)
+title: Taproot Lightning
+speakers: ['Antoine Riard']
+date: 2020-02-06
 transcript_by: Michael Folkson
 translation_by: Blue Moon
 categories: ['conferencia']
 tags: ['taproot', 'lightning', 'schnorr']
 ---
 
-Nombre: Antoine Riard
-
 Tema: A Schnorr-Taproot’ed Lightning
 
 Ubicación: El avance de Bitcoin 
 
-Fecha: 6 de febrero de 2020
-
-Video (acceso de pago): https://www.advancingbitcoin.com/video/a-schnorr-taprooted-lightning,11/
+Video: https://www.advancingbitcoin.com/video/a-schnorr-taprooted-lightning,11/
```

Comment:
no action needed, but noting this isn't valid markdown. It needs <> around the URL. We should clean these up when we add videos to the metadata.